internal/super/flix.go

```diff
@@ -49,6 +49,7 @@ type flixFlags struct {
 	GasLimit    uint64   `default:"1000" flag:"gas-limit" info:"transaction gas limit"`
 	PreFill     string   `default:"" flag:"pre-fill" info:"template path to pre fill the FLIX"`
 	Lang        string   `default:"js" flag:"lang" info:"language to generate the template for"`
+	Networks    []string `default:"" flag:"networks" info:"networks to generate the template for"`
```

Comment:
geesh, good catch. 
User can add 0 or many networks. relied on copilot too much on that one.
updated my PR. Thanks

Reply:
@bthaile are you good with changing the name to something other thna `networks`? Reason is that we already have a pre-existing `network` flag, and this just might add more confusion

Reply:
Yeah because there is a global flag named network, I think it needs to work with that. `networks` is too confusing to someone familiar with the global flags.

Reply:
What about `--exclude-networks`? Default is to run against all networks and anything listed after flag is excluded

Reply:
Changed it to `Targets` I think that is fine

Reply:
Saying "targets" is for "target networks" still holds about the same amount of confusion in my opinion and starts to introduce new patterns that may start to stink into other parts of code. I'd go with what @sisyphusSmiling said, `--exlcude-networks`, as it has the potential to be more applicable across other use cases if something similar is ever needed.